src/renderer/store/modules/Note.js

```diff
@@ -4,12 +4,8 @@ import path from 'path';
 // eslint-disable-next-line import/no-extraneous-dependencies
 import { remote } from 'electron';
 
-const octokit = require('@octokit/rest')({
-  requestMedia: 'application/vnd.github.v3+json',
-  headers: {
-    'user-agent': 'octokit/rest.js v1.2.3',
-  },
-});
+const Octokit = require('@octokit/rest');
+let octokit = Octokit();
```

Comment:
Are you sure?
In their documentation I see:
```
const Octokit = require("@octokit/rest");
const octokit = new Octokit();
```

Reply:
You missed the `new` keyword.

Reply:
I agree.